drf_user/serializers.py

```diff
@@ -362,3 +362,19 @@ def validate(self, attrs: dict) -> dict:
             raise NotFound(_("User with the provided email does not exist."))
 
         return attrs
+
+
+class ImageSerializer(serializers.ModelSerializer):
+    """This serializer is for Image Upload API.
+
+    Params
+    profile_image: OTP received on your email/mobile
```

Comment:
Why OTP received on your email/mobile??

Reply:
Copy pasted the docstring from the previous pull request then forgot to change this description🤦🏻‍♂️